services/QuillLMS/app/controllers/activities_controller.rb

```diff
@@ -62,15 +62,18 @@ def customize_lesson
   end
 
   def activity_session
+    return redirect_to profile_path if !current_user.student?
```

Comment:
Pretty minor Ruby thing, but I think our current pattern is to use `unless` instead of `if !`.  So this could be: `return redirect_to profile_path unless current_user.student?`